python/orca/src/bigdl/orca/automl/search/ray_tune/ray_tune_search_engine.py

```diff
@@ -188,9 +189,10 @@ def run(self):
             search_alg=self._search_alg,
             num_samples=self.num_samples,
             trial_dirname_creator=trial_dirname_creator,
+            callbacks=[CustomProgressCallback()],
```

Comment:
I think this Callback is kind of useless, we can remove it for now. Please correct me if it's critical

Reply:
CustomProgresscallback can avoid unnecessary log output.